### PR TITLE
Add Build dependencies to app setup instructions

### PR DESCRIPTION
**Pull request by llogiq**
(I choose the semantic branch as this is where the interesting stuff is going to go on. If I should backport to main, please ping me)

### CODE REVIEW

File: apps/desktop/README.md [hunk 1]
```diff
@@ -2,6 +2,25 @@
...
+* glib
+* gtk3 including webkit-gtk
+* npm (usually in the node package)
+* openBLAS
+* pango
+* pnpm
+* protobuf
+* rustup
+* wget
+
```
Comment (ggordonhall) [inline apps/desktop/README.md:23]:
```suggestion
## Dependencies

To build the Tauri app you need the following dependencies:
- `pnpm`
- `rustup`
- `clang` `cmake` `wget`
- `openBLAS`

Linux users need to ensure that the following are present:
- `AppImageKit`
- `atk`
- `dbus`
- `glib` `gtk3` (including `webkit-gtk`)
- `pango`
- `protobuf`
 
```

File: apps/desktop/README.md [hunk 2]
```diff
@@ -19,4 +38,6 @@ npm run build-app
 Alternatively, to run the app in dev mode:
 ```
 npm run start-app
-```
\ No newline at end of file
+```
+
+If the build should fail, please delete all `node_modules` directories you may find and try again. `npm` is sometimes obtuse with caches.
```
Comment (ggordonhall) [inline apps/desktop/README.md:43]:
```suggestion
If the build fails, delete all `node_modules` directories and try again.
```

**Review by ggordonhall — COMMENTED** [review]
Thanks for this. Left a couple of edits.

**Comment by llogiq** [comment]
Thanks, @ggordonhall

**Review by ggordonhall — APPROVED** [review]

File: apps/desktop/README.md [hunk 3]
```diff
@@ -2,6 +2,22 @@
...
+- `clang` `cmake` `wget`
+- `openBLAS`
+
+Linux users need to ensure that the following are present:
+- `AppImageKit`
+- `atk`
+- `dbus`
+- `glib` `gtk3` (including `webkit-gtk`)
+- `pango`
+- `protobuf`
```
Comment (rsdy) [inline apps/desktop/README.md:19]: I think we need `protobuf` always, not just on Linux.

**Review by rsdy — APPROVED** [review]

**Pull request merged**